float64 muscle1
float64 muscle2
float64 muscle3
float64 muscle4